src/main/java/com/commercetools/sync/services/impl/BaseService.java

```diff
@@ -28,6 +32,7 @@
     final Map<String, String> keyToIdCache = new ConcurrentHashMap<>();
 
     private static final String CREATE_FAILED = "Failed to create draft with key: '%s'. Reason: %s";
+    private static final int MAXIMUM_ALLOWED_UPDATE_ACTIONS = 500;
```

Comment:
Default is good but would be nice if one could override it in syncOptions.

Reply:
You mean have it as one of the options in the sync? What are the use cases?